packages/framework-core/src/booster.ts

```diff
@@ -37,6 +37,10 @@ export class Booster {
     configurator(this.config)
   }
 
+  public static getConfiguration(): BoosterConfig {
+    return this.config
+  }
+
```

Comment:
What I meant is to remove this getter, and make the `config` getter like this:
```typescript
  *public* static readonly config = new BoosterConfig(checkAndGetCurrentEnv())
```

Reply:
done!!